Fix for issue #3754

michelvosje opened the pull request:
These changes fix the issue reported in #3754.

I've used the following codes to test the changes and they all worked.
```
-- Normal SQL code.
select
	id,
	name,
	age
from public.test;

-- Clipboard content will now be seen as source code.
"select
	id,
	name,
	age
from public.test;"

-- Line endings within SQL code.
"select \n" +
"	id, \n" +
"	name, \n" +
"	age \n" +
"from public.test;"

-- Line comment
"select \n" +
"	id, \n" +
//"	name, \n" +
"	age \n" +
"from public.test;"

-- Block comment
"select \n" +
/*"	id, \n" +
"	name, \n" +*/
"	age \n" +
"from public.test;"

-- Comment characters within line comment
"select \n" +
"	id, \n" +
//"*/	name, \n"
"	age \n" +
"from public.test;"

-- Comment characters within block comment
"select \n" +
/*"	id, \n" +
"*\/name, \n" +*/
"	age \n" +
"from public.test;"
```

michelvosje committed:
#3754 Improved transformation of source code when pasting SQL code.

When pasted SQL code is within double quotes, the surrounding source code was incorrectly filtered.
This change makes sure the source code comments are properly removed.
It now has support for Java style line and block comments.

michelvosje committed:
Merge branch 'devel' of https://github.com/michelvosje/dbeaver into devel

serge-rider commented:
Thanks, this looks great.
I'll test it and if everything will work fine then it will be included in 5.1.6

michelvosje committed:
#2768 Removed auto unindent on new line after delimiter.

michelvosje commented:
Added a commit to fix issue #2768.
I removed the auto unindentation because it doesn't work properly.

Sorry i am not really used to creating pull requests on GitHub. So its added to the same pull request as the previous commit. Should i have made a ticket-branch in my fork?

serge-rider commented:
It is better to make two separate PRs because these fixes seems to be unrelated.
